ansi.py

```diff
@@ -70,15 +77,39 @@ def syntax_change(self, view):
 
 
 class AnsiColorBuildCommand(Default.exec.ExecCommand):
-    def on_finished(self, proc):
-        super(AnsiColorBuildCommand, self).on_finished(proc)
 
+    process_on_data = False
+    pricess_on_finish = True
+
+    @classmethod
+    def update_build_settings(cls):
```

Comment:
Nice! This is much better than processing the setting on every single build output change.